configure.ac

```diff
@@ -75,12 +75,7 @@ SP_CHECK_CXXFLAGS_APPEND([OUR_CXXFLAGS], [\
 ##########################################################################
 # check VMA extra API
 #
-AC_ARG_ENABLE(
-    [vma-api],
-    AC_HELP_STRING([--enable-vma-api],
-                   [SOCKPERF: enable vma extra api support (default=no)]),
-    [have_vma_api=$enableval],
-    [have_vma_api=no])
+have_vma_api=yes
 AS_IF([test "x${have_vma_api}" == "xyes"],
     [AC_CHECK_HEADER([mellanox/vma_extra.h],
         [AC_DEFINE([USING_VMA_EXTRA_API],[1],[[Enable using VMA extra API]])],
```

Comment:
@igor-ivanov please also remove
AC_MSG_CHECKING(
    [for vma extra api])
AC_MSG_RESULT([${have_vma_api}])
and the AS_IF
so we will have only 
AC_CHECK_HEADER([mellanox/vma_extra.h],
    [AC_DEFINE([USING_VMA_EXTRA_API],[1],[[Enable using VMA extra API]])],
    [AC_MSG_NOTICE([vma_extra.h file not found])]
    [have_vma_api=no])

Reply:
I would like to keep one as is because from my point it provides more friendly output for end user

Reply:
ok

Reply:
my opinion change to default true